.gitlab-ci.yml

```diff
@@ -41,3 +63,51 @@ test:extra-tools:release-tool:
   script:
     # Run release-tool unit tests.
     - python3 -m pytest extra/test_release_tool.py
+
+test:staging:backend-tests:
+  image: debian:buster
+  stage: test
+  timeout: 4h
+  only:
+    variables:
+      - $RUN_TESTS_STAGING == "true"
+
+  services:
+    - docker:dind
```

Comment:
So we managed to make it work with DinD? This is great! I would love to understand "why", so that maybe we can rework mender-qa pipeline with that knowledge.

Reply:
yes, it works using dind as a service because I log in to AWS/kubectl from the inner container, it works just fine without mounting the volumes.

Reply:
Ah, of course. Got it.